Graphics/HLSL2GLSLConverterLib/include/GLSLDefinitions.h

```diff
@@ -34,9 +34,9 @@
 // From GLES 3.1 spec:
```

Comment:
Let's not touch this file - it only adds a lot of changes to the PR without real need.

Reply:
Not sure how this happened, will remove.